cnf-tests/Dockerfile

```diff
@@ -46,6 +38,12 @@ RUN go build -mod=vendor -o /oslat-runner oslat-runner/main.go && \
     go build -mod=vendor -o /cyclictest-runner cyclictest-runner/main.go && \
     go build -mod=vendor -o /hwlatdetect-runner hwlatdetect-runner/main.go
 
+# build latency testing suite
```

Comment:
If the image must contain only the latency test binary, we should move all the utilities required by the other tests (sctptester, strestest, podutils) to another image that can be used by the tests.

Not blocking this PR but we should file an item for the followup.

Reply:
I'll file a ticket for this.